### PR TITLE
Enable Arm32 Test Build

### PR DESCRIPTION
**Pull request by gkhanna79**
@jkotas @ramarag PTAL

### CODE REVIEW

File: tests/src/Interop/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,12 @@
 
+if(WIN32)
+	if(CLR_CMAKE_HOST_ARCH STREQUAL arm)
+	    list(APPEND LINK_LIBRARIES_ADDITIONAL
```
Comment (ramarag) [inline tests/src/Interop/CMakeLists.txt]: NIt

I don't see LINK_LIBRARIES_ADDITIONAL being defined anywhere is it a pre defined property in cmake ?

**Comment by ramarag** [comment]
LGTM otherwise

File: tests/src/Interop/CMakeLists.txt (hunk 1 above)
Comment (gkhanna79) [inline tests/src/Interop/CMakeLists.txt]: No, it is a custom variable. I believe CMake will define it if it is not already defined.

File: tests/setup-runtime-dependencies.cmd [hunk 2]
```diff
@@ -35,6 +35,12 @@ goto Usage
 if not defined __OutputDir goto Usage
 if not defined __Arch goto Usage 
 
+REM Check if the platform is supported
+if /i %__Arch% == "arm" (
+    echo No runtime dependencies for Arm32.
+    exit /b 0
+    )
```
Comment (rahku) [inline tests/setup-runtime-dependencies.cmd]: could you also add the same condition for arm64. It fails to restore the package for arm64 currently as it is not available. This does not fail the build though

**Pull request merged**